### PR TITLE
frontends.jax.numpy: add full func

### LINKED ISSUE #9977: full

### PR DESCRIPTION
**Pull request by hmahmood24**
Close #9977

### CODE REVIEW

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.

**Comment by hmahmood24** [comment]
Commenting to prevent this PR from turning stale!

**Comment by RickSanchezStoic** [comment]
I don't see any errors related to this test, so looks good to me.
